### ISSUE TITLE
Does not install format.cc

### ISSUE THREAD

**Issue by WscriChy** [cppformat/cppformat#293]
Why release 2.0.1 does not install neither format.cc, nor posix.h?

**Comment by vitaut** [comment]
I've updated the CMake files to https://github.com/cppformat/cppformat/tree/7ee287d3d9e340737f3ad66292e7bc60a53b3053 which should have `find_package` and export support, but doesn't have improvements introduced after this commit because in the further commits backwards compatibility is broken by moving the code to the `cppformat` subdirectory. Breaking changes will have to wait until a major release.

`format.cc` is not used in the default configuration and `posix.h` implements undocumented experimental API so it's OK to omit them for now.

**Comment by WscriChy** [comment]
I mean how header-only mode is supposed to work if `format.cc` is not present.

```
#ifdef FMT_HEADER_ONLY
# include "format.cc"
#endif
```

**Comment by vitaut** [comment]
It won't work unfortunately, so I've integrated more changes from `master` into the [2.x branch](https://github.com/cppformat/cppformat/tree/2.x). This brings the CMake config up to date with `format.cc` and `posix.h` included in installation. As a side effect the code has been moved to the `cppformat` directory and including from the top-level directory is deprecated (but works). @WscriChy, if you don't see any other issues, I can tag this as 2.1.0.
